packages/devtools_app/lib/src/screens/debugger/call_stack.dart

```diff
@@ -63,40 +63,42 @@ class _CallStackState extends State<CallStack>
     final frameDescription = frame.description;
     final locationDescription = frame.location;
 
-    if (asyncMarker) {
-      child = Row(
-        children: [
-          const SizedBox(width: defaultSpacing, child: Divider()),
-          Padding(
-            padding: const EdgeInsets.symmetric(horizontal: densePadding),
-            child: Text(
-              frameDescription,
-              style: selected ? theme.selectedTextStyle : theme.subtleTextStyle,
+    child = asyncMarker
```

Comment:
@kenzieschmoll what do you think about cases like this? Is this cleaner or more obfuscated?

Reply:
I find it harder to read when one or more of the three operands need to be split across multiple lines.

Reply:
I do think the if / else block is a little bit easier to read here given how many lines each operand is.

Reply:
Agreed. Reverted this one